engine/env/spark/src/main/java/org/datacleaner/spark/SparkJobLifeCycleListener.java

```diff
@@ -36,21 +36,21 @@
      * Triggered when a node starts processing a task. This will be executed
      * on the nodes themselves.
      */
-    void onPartitionProcessingStart();
+    void onPartitionProcessingStart(SparkJobContext sparkJobContext);
```

Comment:
Could we add the partition index and the number of partitions as arguments here? Only if it's low-hanging fruit ...